wagtail/admin/templates/wagtailadmin/pages/index.html

```diff
@@ -117,7 +119,9 @@
                         url += `?position=${(index)}`;
                         let CSRFToken = $('input[name="csrfmiddlewaretoken"]', orderform).val();
                         $.post(url, {csrfmiddlewaretoken: CSRFToken}, function(){
-                            addMessage('success', `"${$(currentlySelected).data('page-title')}" has been moved successfully from ${currentPosition +1} to ${index +1}.`)
+                            const text = `"${$(currentlySelected).data('page-title')}" has been moved successfully from ${currentPosition + 1} to ${index + 1}.`;
+                            const event = new CustomEvent('w-message:add', { detail: { status: 'stauts', text, clear: true } });
```

Comment:
```suggestion
                            const event = new CustomEvent('w-messages:add', { detail: { status: 'success', text, clear: true } });
```

Sorry, not sure if this was from my previous suggestion, but this did not work, needs to be `messages` plural`, additionally the status should be success.

Reply:
Strange I guess it's my fault, during the finale check for default behavior I must have messed it up. Although I checked everything, cool I would fix that up